### ISSUE TITLE
auto append tribe url param to app url #181

### ISSUE THREAD

**Issue by tomastiminskas** [stakwork/sphinx-kotlin#572]
When opening the app url from a tribe, automatically append a tribe url param with the tribe uuid. This will help apps like Second Brain identify the tribe context

Remember, any url can be set as the app url... so we need to make sure to append it properly, because the app url could already have url params set. So you can check if there is a "?" in the app url and if so append &tribe={UUID}, otherwize append ?tribe={UUID}

**Comment by ariel10aguero** [comment]
Do you mean the FeedItemLink which has this regex? 

`"sphinx\\.chat:\\/\\/\\?action=share_content&feedURL=.*"`

**Comment by tomastiminskas** [comment]
No, this is for a feature it's not implemented in android yet. So skip it for now
